src/benchmarks/micro/corefx/System.Linq/Perf.Linq.cs

```diff
@@ -145,6 +145,31 @@ public void Skip(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType
         public void Take(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.Take(size - 1), _consumer);
 
+#if !NETFRAMEWORK
+        [Benchmark]
+        [ArgumentsSource(nameof(IterationSizeWrapperData))]
+        public void TakeLast(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
+            => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.TakeLast(size - 1), _consumer);
+
+        [Benchmark]
+        [ArgumentsSource(nameof(IterationSizeWrapperData))]
+        public int[] TakeLastToArray(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
+        {
+            IEnumerable<int> source = Perf_LinqTestBase.Wrap(_sizeToPreallocatedArray[size], wrapType);
+
+            return source.TakeLast(size - 1).ToArray();
```

Comment:
`ToArray()` is a very expensive operation which would dominate the reported time for this benchmark, please remove the `TakeLastToArray` benchmark, what you wrote in `TakeLast` is enough

Reply:
Generally the source in the test is already IList so there indeed there is no need to make separate for ToArray/ToList, the NoWrap test shows enough difference.

Done